File: 프로그래머스/0/120829. 각도기/각도기.py
def solution(angle):

    if angle < 90:
        return 1
    elif angle == 90:
        return 2
    elif 90 < angle and angle < 180:
        return 3
    else:
        return 4;

# if문 (if, elif, else)
# 주의 사항
# 1. 들여쓰기 주의: 파이썬은 들여쓰기로 코드 블록을 인식함
# 2. if문의 조건 사용 후 콜론(:)을 필수적으로 붙여야 함

# 한 줄 if문  (삼항 조건 연산자)
# 변수 = 값1 if 조건 else 값2

File: 프로그래머스/0/120829. 각도기/각도기.java
class Solution {
    public int solution(int angle) {
        
        if(angle < 90) {
            return 1;
        }else if(angle == 90) {
            return 2;
        }else if(90 < angle && angle < 180) {
            return 3;
        }else {
            return 4;
        }
    }
}

// (참고) 삼항연산자를 적용하는 경우
// return angle < 90 ? 1 : angle == 90 ? 2 : angle == 180 ? 4 : 3;
